doc/policies/trees.hpp

```diff
@@ -47,12 +47,13 @@ are nearby should lie in similar nodes.
 We can rigorously define what a tree is, using the definition of **space tree**
 introduced in the following paper:
 
+@code
 @quote
 R.R. Curtin, W.B. March, P. Ram, D.V. Anderson, A.G. Gray, and C.L. Isbell Jr.,
 "Tree-independent dual-tree algorithms," in Proceedings of the 30th
 International Conference on Machine Learning (ICML '13), pp. 1435--1443, 2013.
 @endquote
-
+@endcode
```

Comment:
```suggestion
@endcode

```

(Just to keep an empty line between them.)